examples/react-native/RunAnywhereAI/ios/Podfile

```
@@ -77,20 +77,27 @@ target 'RunAnywhereAI' do
-          # Remove the invalid -G flag and set preprocessor definitions properly
-          config.build_settings['OTHER_CFLAGS'] = '$(inherited)'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)']
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'HAVE_INTTYPES_H=1'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'HAVE_PKCRYPT=1'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'HAVE_STDINT_H=1'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'HAVE_WZAES=1'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'HAVE_ZLIB=1'
-          config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] << 'MZ_ZIP_NO_SIGNING=1'
+          # Completely override compiler flags to remove -G
+          cflags = config.build_settings['OTHER_CFLAGS'] || ''
+          cflags = cflags.gsub(/-G\s*/, '') if cflags.is_a?(String)
```

RuboCop (1.82.1)

[warning] Useless assignment to variable - `cflags`.

(Lint/UselessAssignment)